wire/hybridbus.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wire
+
+import (
+	"context"
+	"fmt"
+
+	"polycry.pt/poly-go/errors"
+	"polycry.pt/poly-go/sync/atomic"
+)
+
+var _ Bus = (*hybridBus)(nil)
+
+// A Bus that can send messages over a local bus and a remote bus.
```

Comment:
```suggestion
// A Bus that uses multiple buses when trying to send a message.
// Could be used with a local bus and a remote bus.
```

Reply:
I prefer @RmbRT 's version. It's clearer in describing what this object provides. Your suggestion focuses on how it's implemented, which is not so relevant here.